tests/notifications.test.ts

```
@@ -11,95 +13,231 @@ import express from "express";
+      installationId: genericTestInstallationId1,
+    });
+    await notificationClient.deleteInstallation({
+      installationId: genericTestInstallationId2,
+    });
+    await notificationClient.deleteInstallation({
+      installationId: genericTestInstallationId3,
+    });
+    await notificationClient.deleteInstallation({
+      installationId: genericTestInstallationId4,
+    });
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)